xbmc/cores/VideoPlayer/Process/gbm/VideoBufferDRMPRIME.h

```diff
@@ -61,7 +61,7 @@ class IVideoBufferDRMPRIME : public CVideoBuffer
   uint32_t m_handles[AV_DRM_MAX_PLANES] = {};
 
 protected:
-  explicit IVideoBufferDRMPRIME(int id);
+  explicit IVideoBufferDRMPRIME(int id) : CVideoBuffer(id) { }
```

Comment:
Can you still consider this an interface if it derives from a (non-interface) class?

Reply:
IVideoBufferDRMPRIME is an abstract class and from my point of view makes it an interface, I guess we can rename it to CVideoBufferDRM and duplicate code if that makes it better?